cryptofeed/feed.py

```diff
@@ -94,8 +94,9 @@ def __init__(self, address: Union[dict, str], timeout=120, timeout_interval=30,
         self.origin = origin
         self.checksum_validation = checksum_validation
         self.ws_defaults = {'ping_interval': 10, 'ping_timeout': None, 'max_size': 2**23, 'max_queue': None, 'origin': self.origin}
-        self.key_id = os.environ.get(f'CF_{self.id}_KEY_ID') or self.config[self.id.lower()].key_id
-        self.key_secret = os.environ.get(f'CF_{self.id}_KEY_SECRET') or self.config[self.id.lower()].key_secret
+        self.subaccount = subaccount
```

Comment:
same comments here. Honestly I hate the env vars, they are likely to go away. You should move this all to the FTX class (the subaccount specifics)